controller/env/broker.go

```diff
@@ -75,15 +75,8 @@ func NewBroker(ae *AppEnv, synchronizer RouterSyncStrategy) *Broker {
 
 func (broker *Broker) RouterConnected(router *network.Router) {
 	go func() {
-
 		log := pfxlog.Logger().WithField("routerId", router.Id).WithField("routerName", router.Name).WithField("routerFingerprint", router.Fingerprint)
 
```

Comment:
Will this allow existing connections to replaced? If so, is that desirable? If so, should we close the old connection?

Reply:
It can't be triggered w/ an existing connection based on how the fabric code reads and behaves.

I created a router with the same identity but two different sets of ports they listen on. The fabric refused to allow the second instance to run. Attempting to use the same configuration on the same machine causes port usage errors.

Reply:
Ah, gotcha. Makes sense to remove if it's redundant